rlkit/torch/sac/sac.py

```diff
@@ -293,3 +328,26 @@ def get_epoch_snapshot(self, epoch):
             task_enc=self.policy.task_enc.state_dict(),
         )
         return snapshot
+
+
+# TODO: make a new file to place these utils, maybe torch/sac/sac_utils.py? alternatively just move them into pytorch utils
```

Comment:
I agree these should go to some util file, pytorch_utils seems fine

Reply:
see 933d38